doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

```diff
@@ -332,13 +334,14 @@ Hence, the GNSS module needs to download the Ephemeris data from the satellite b
 .. note::
    |gps_tradeoffs|
 
-nRF Cloud A-GPS compared with SUPL library
-------------------------------------------
+nRF Cloud compared with SUPL library
+------------------------------------
 
-The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
-Another advantage of nRF Cloud A-GPS library is that the data is encrypted whereas SUPL uses plain socket.
-Also, no licenses are required from external vendors to use nRF Cloud A-GPS, whereas for commercial use of SUPL, you must obtain a license.
-The :ref:`lib_nrf_cloud_agps` library is also highly integrated into `Nordic Semiconductor's IoT cloud platform`_.
+* The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
+* With nRF Cloud, the data is encrypted whereas SUPL uses plain socket.
+* nRF Cloud supports assistance also for QZSS satellites, SUPL only for GPS.
```

Comment:
```suggestion
* nRF Cloud also supports assistance for QZSS satellites. SUPL applies only to GPS.
```

Reply:
As discussed, I'll make it "nRF Cloud also support assistance for QZSS satellites, while SUPL is limited to GPS".